server/master/src/main/java/org/apache/accumulo/master/Master.java

```diff
@@ -192,6 +193,7 @@
   private ReplicationDriver replicationWorkDriver;
   private WorkDriver replicationWorkAssigner;
   RecoveryManager recoveryManager = null;
+  private AtomicLong replicationLatency = new AtomicLong(0l);
```

Comment:
Personal preference: use upper-case `L` for Long literals. It's much easier to read (this looks like `01`).